app/views/catalog/record/_marc_contents_summary.html.erb

```diff
@@ -42,17 +42,14 @@
   </dd>
 <% end %>
 
-<% summary = document.fetch(:summary_struct, []).first || {} %>
-<% if summary[:fields].present? || summary[:unmatched_vernacular].present? %>
-  <dt>Summary</dt>
-  <% if summary[:fields].present? %>
-    <% summary[:fields].each do |field| %>
-      <dd>
-        <%= render_struct_field_data document, field[:field] %>
-        <%= "<br/>#{field[:vernacular]}".html_safe unless field[:vernacular].nil? %>
-        <%= "<br/>".html_safe if field != summary[:fields].last %>
-      </dd>
-    <% end %>
+<% document.fetch(:summary_struct, []).each do |summary|%>
+  <% summary.fetch(:fields, []).each do |field| %>
+    <dt><%= summary[:label] %></dt>
```

Comment:
```suggestion
    <dt><%= summary[:label] || 'Summary' %></dt>
```

Is there any sense falling back on a default label, or do we know all summaries have labels already?

Reply:
They do always have a label set in searchworks_traject_indexer